src/Integration.Vsix/SonarLintNotificationsPackage.cs

```diff
@@ -62,7 +61,8 @@ protected override void Initialize()
             var sonarqubeService = this.GetMefService<ISonarQubeService>();
 
             notifications = new SonarQubeNotificationService(sonarqubeService,
-                new NotificationIndicatorViewModel(), new TimerWrapper { Interval = 60000 }, output);
+                new NotificationIndicatorViewModel(), new TimerWrapper { Interval = 60000 },
+                this.GetMefService<ISonarLintOutput>());
```

Comment:
Let's store it into some variable instead of asking for it here and later on.

Reply:
I was not sure if `Initialize()` is always called **before** `OnLoadOptions()`. Probably it is safe, but I could not find documentation about the order of invocation. On the other hand, the exception should not be thrown very often and the service resolution should not put performance penalty.

Reply:
_Initialize_ is called as part of the package being "sited" (i.e. started and hooked in to VS - the underlying native interface method is called _SetSite_). It **should** happen before any other methods get called.